eessi/testsuite/common_config.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+perflog_format = '|'.join([
+    '%(check_job_completion_time)s',
+    '%(osuser)s',
+    '%(version)s',
+    '%(check_unique_name)s',
+    '%(check_info)s',
+    '%(check_system)s',
+    '%(check_partition)s',
+    '%(check_environ)s',
+    '%(check_exclusive_access)s',
+    '%(check_num_tasks)s',
+    '%(check_num_cpus_per_task)s',
+    '%(check_num_tasks_per_node)s',
+    '%(check_num_gpus_per_node)s',
+    '%(check_use_multithreading)s',
+    '%(check_modules)s',
+    '%(check_jobid)s',
+    '%(check_perfvalues)s',
+])
+
+format_perfvars = '|'.join([
+    '%(check_perf_var)s',
+    '%(check_perf_value)s',
+    '%(check_perf_lower_thres)s',
+    '%(check_perf_upper_thres)s',
+    '%(check_perf_unit)s',
+    ''  # final delimiter required
+])
+
+
+def common_logging_config(prefix=None):
+    """
+    return default logging configuration as a list: stdout, file log, perflog
+    :param prefix: file log prefix
+    """
+    prefix = os.getenv('RFM_PREFIX', prefix if prefix else '.')
+
+    os.makedirs(os.path.join(prefix, 'log'), exist_ok=True)
+
+    return [{
+        'level': 'debug',
+        'handlers': [
+            {
+                'type': 'stream',
+                'name': 'stdout',
+                'level': 'info',
+                'format': '%(message)s',
+            },
+            {
+                'type': 'file',
+                'name': os.path.join(prefix, 'log', 'reframe.log'),
```

Comment:
Bikeshedding, I admit, but... I'd probably call it `logs` for consistency (the performance logs end up in `perflogs`, which also uses the plural)

Reply:
done in https://github.com/EESSI/test-suite/pull/80/commits/6c290a19b4079f3c158aecaddd0353791c0189c2